website/docs/language/stacks/reference/tfstacks-cli.mdx

```diff
@@ -7,6 +7,10 @@ description: The terraform-stacks-cli is a command-line tool for validating, ini
 
 The `terraform-stacks-cli` is a command-line tool for validating, initializing, and testing Stack configurations.
 
+## Requirements
+
+The `terraform-stacks-cli` requires an `alpha` version of Terraform, and you must use at least version `terraform_1.10.0-alpha20241009` or higher. You can download an `alpha` version of Terraform on the [releases page](https://releases.hashicorp.com/terraform/).
```

Comment:
```suggestion
The `terraform-stacks-cli` requires Terraform version `1.10.0-alpha20241009` or higher. You can download an `alpha` version of Terraform on the [releases page](https://releases.hashicorp.com/terraform/).
```

I'm not exactly sure the best way to phrase this but to me saying that you need _an_ alpha version of terraform, then listing the minimum version is a bit confusing IMO since there's lots of alpha versions of Terraform :) Just my two cents and feel free to reject if you disagree!

Reply:
You actually **do** need an alpha version, it's weird. Does this say "you need to use an alpha version AND that version is at least `terraform_1.10.0-alpha20241009` or higher? (I'm using the `terraform_1.10.0-alpha20241009` name in case folks do a CTRL+F) 

I could also try rephrasing as:
```suggestion
The `terraform-stacks-cli` requires:
1. an `alpha` version of Terraform
2. At least version `terraform_1.10.0-alpha20241009` or higher
You can download an `alpha` version of Terraform on the [releases page](https://releases.hashicorp.com/terraform/). We recommend downloading the latest alpha version of Terraform to use the most up-to-date functionality.
```

I'm down to do whichever makes most sense to you!

Reply:
I see! Sorry, I wasn't aware of that requirement. I think if that's the case the origional wording makes most sense :)